client/src/constants.ts

```diff
@@ -1 +1,2 @@
-export const HOSTNAME = '127.0.0.1:3001';
+// export const HOSTNAME = '127.0.0.1:3001';
```

Comment:
Wanna just delete this?

Reply:
Actually, we will probably want to keep the hostname as the local version. I have to change it to a new IP when testing on my windows machine to point to my `docker-machine ip`